packages/blaze/backcompat.js

```diff
@@ -1,3 +1,6 @@
+/* global Blaze UI Handlebars ReactiveVar */
```

Comment:
We can drop `UI` as it's deprecated

Reply:
With this change, many compatibility tests present in old_template_test.js will be deleted, creating possible breaking chnages. Is that what we want?

Reply:
Yes, for 3.0 we will likely introduce breaking changes. Note that it should only be breaking for the romeval of `UI`

Reply:
Thank you, I remove it